lib/player.rb

```diff
@@ -0,0 +1,16 @@
+require 'pry'
+require_relative './deck'
+
+class Player
+    attr_reader :deck, :name
+
+    def initialize(name, deck)
+        @name = name
+        @deck = deck 
+    end 
+
+    def has_lost?
+        return true if @deck.cards.none?
```

Comment:
I love the use of explicit and implicit `return`s here. One thing to keep in mind is that `none?` is a predicate/boolean method and therefore we know that it will return either `true` or `false` after it is called, we can use that to our advantage and rewrite the code like so:

```ruby
def has_lost?
  @deck.cards.none?
end
```

The result will either be `true` or `false`.